tests/test-timber-term.php

```diff
@@ -66,39 +70,41 @@ function testConstructor() {
 
 		function testTerm() {
 			$term_id = $this->factory->term->create();
-			$term = new Timber\Term($term_id);
+			$term = Timber::get_term($term_id);
 			$this->assertEquals('Timber\Term', get_class($term));
 		}
 
 		function testGetTermWithObject() {
 			$term_id = $this->factory->term->create(array('name' => 'Famous Commissioners'));
 			$term_data = get_term($term_id, 'post_tag');
 			$this->assertTrue( in_array( get_class($term_data), array('WP_Term', 'stdClass') ) );
-			$term = new Timber\Term($term_id);
+			$term = Timber::get_term($term_id);
 			$this->assertEquals('Famous Commissioners', $term->title());
 			$this->assertEquals('Timber\Term', get_class($term));
 		}
 
 		function testTermConstructWithSlug() {
 			$term_id = $this->factory->term->create(array('name' => 'New England Patriots'));
-			$term = new Timber\Term('new-england-patriots');
+			$term = Timber::get_term($term_id);
```

Comment:
Is the idea here that we don’t allow instantiation via slug, right? Shouldn’t we have to delete that test then, or make sure it’s throwing a doing-it-wrong message? Because the name of the test suggests that it’s specifically about constructing a term with a slug.

Reply:
Yep, I moved this into a pending test to look at as part of #2087.